src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebProxy.cs

```diff
@@ -18,6 +18,32 @@ internal WebProxy(Uri address)
             _proxyAddress = address;
         }
 
+        public override bool Equals(object obj)
+        {
+            if (obj == null || GetType() != obj.GetType())
```

Comment:
I guess the type comparison is covered in line 28. 
```suggestion
            if (obj == null)
```

Reply:
Type comparison and null check are both covered in the Equals() so I've just forwarded the function directly after casting as WebProxy.